Documentation/toc.yml

```diff
@@ -96,20 +95,32 @@
         href: Diagnostics/ConfiguringDiagnostics.md
       - name: Using the Visual Profiler
         href: Diagnostics/UsingVisualProfiler.md
-  - name: Other Features
+    - name: Extension Service Creation Wizard
+      href: Utilities/ExtensionServiceCreationWizard.md
+  - name: Experimental Features
     items:
+    - name: Spatial Understanding
+      href: TODO.md
     - name: QR Tracking
       href: TODO.md
     - name: Spatial Audio
       href: TODO.md
   - name: Advanced topics
     items:
-    - name: Performance
-      href: Performance/PerfGettingStarted.md
     - name: Shared Experiences
       href: TODO.md
-    - name: Moving your project from HTK to MRTK
-      href: HTKToMRTKPortingGuide.md
+- name: How-to 
```

Comment:
Unless someone is actually going to do this in the next few days, I recommend not adding new TODOs.

These are very frustrating to read (as a consumer) and are easy to forget about (as a producer)

Reply:
I'm going to go ahead and merge this PR as-is to get it in, then add this to my incoming docs PR.